lambda/js/src/main/scala/feral/lambda/facade/Context.scala

```diff
@@ -42,7 +42,7 @@ private[lambda] sealed trait CognitoIdentity extends js.Object {
 @js.native
 private[lambda] sealed trait ClientContext extends js.Object {
   def client: ClientContextClient = js.native
-  def Custom: js.Any = js.native
+  def custom: js.UndefOr[js.Any] = js.native
```

Comment:
I'm 99% sure this was a typo in the Typescript. I don't know why it would be different for Node.js, or why it would be capitalized in general.